<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COCSO VISION  </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="recordGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>601</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>显示面板</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="videoCanvas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>570</width>
       <height>441</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);
border-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgba(53, 57, 59, 128);</string>
     </property>
     <property name="text">
      <string>显示区域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="calibrateButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>490</y>
       <width>100</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>校准</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/calibrate1.png</normaloff>:/icons/calibrate1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>100</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/eye.png</normaloff>:/icons/eye.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="recordButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>490</y>
       <width>100</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>跟踪</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/tracking.png</normaloff>:/icons/tracking.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="testButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>490</y>
       <width>100</width>
       <height>54</height>
      </rect>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/video.png</normaloff>:/icons/video.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>181</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>参数面板</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="sameID">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>155</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>同一个人</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="sameIDtextBrowser">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rMaxHorizontalSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-2</number>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="radiusMax">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>最小人脸</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="similarityLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>相似度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="culrrentRadius">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>尺寸</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="similaritytextBrowser">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>105</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="design.qrc"/>
 </resources>
 <connections/>
</ui>
